apiary.apib

```diff
@@ -758,6 +766,8 @@ You may create your own question using this action. It takes a JSON
 object containing a question and a collection of answers in the
 form of choices.
 
+You need ROLE_ADMIN to access this resource.
```

Comment:
Just though that we should add ROLEs explanation in the spec. Obviously customer token is not allowed to POST here.